front/src/components/signup.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from "react";
+
+export default class SignUp extends Component {
+    constructor(props){
+        super(props);
+        this.state ={
+            firstName: '',
+            lastName: '',
+            email: '',
+            password: ''
+        }
+        this.signUp = this.signUp.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleChange(event) {
```

Comment:
`    handleChange(event) {
        const {name, value} = event.target;
        this.setState({[name]: value});
    }`